STYLE-GUIDE.md

```diff
@@ -2,22 +2,41 @@
 
 This document acts as a Style Guide for the language and wording used in exercises.
 
-## General principle
+## Application
 
-All content should be written in US English.
+These rules should be followed in all exercises. 
+Existing descriptions and test-cases can be updated to adhere to them without requiring "replacement" test cases.
+
+### Consistency within an exercise.
+
+There are some terms that have multiple valid spellings (e.g. "lower case" vs "lowercase"). 
+Where a consistent style has not been agreed within this document, these must be consistent within an exercise.
+
+### Exceptions
+
+Exercises may vary from these rules if there is general consensus across maintainers that this is reasonable. 
+For example, an exercise about converting between different units might choose not to use SI measurements.
+
+## Language
+
+All content should be written in US English. In the future other translations may occur, but the "official" Exercism language is US English.
+
+### Measurements
 All units of measurement must be [SI](https://en.wikipedia.org/wiki/International_System_of_Units) or [SI-derived](https://en.wikipedia.org/wiki/SI_derived_unit) units.
-In the future other translations may occur, but the "official" Exercism language is US English.
 
-## Explain or substitute mathematical and esoteric terms.
+### Abbreviations, acronyms and initialisms
+> All abbreviations, acronyms and initialisms should be defined. 
+> - If the unabbreviated term is clear, always include the full wording at first use (e.g. "Domain-specific language (DSL)");
```

Comment:
What does "If the unabbreviated term is clear" mean? The term "Domain" in this case is jargon *and* ambiguous in the context of programming languages. It's term also doesn't _explain_ what it is. So what is the rule exactly here?

Reply:
I had the same question :) The example seems to me an application of the rule above it ("All abbreviations, acronyms and initialisms should be defined.")